frontend/src/Series/Index/Overview/SeriesIndexOverviewInfo.js

```diff
@@ -254,6 +256,7 @@ SeriesIndexOverviewInfo.propTypes = {
   sizeOnDisk: PropTypes.number,
   sortKey: PropTypes.string.isRequired,
   showRelativeDates: PropTypes.bool.isRequired,
+  longDateFormat: PropTypes.string.isRequired,
```

Comment:
oops, i dont think this line is required.. this line needs to be removed... i think..

Reply:
Yeah, it's not being used, or passed in.